aws/cf_template.go

```diff
@@ -24,6 +24,13 @@ const (
 	//
 	// [0]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-listenerrule.html#cfn-elasticloadbalancingv2-listenerrule-priority
 	internalTrafficDenyRulePriority int64 = 1
+
+	// LoadBalancerResou  rceLogicalID is the logical ID of the LoadBalancer
+	// resource in the CloudFormation template.
+	//
+	// !!!Caution
```

Comment:
we don't use such comments
Every comment should be only there if it's not obvious from code.
So I would be in favor of dropping this line

Reply:
removed the `!!!Caution` part and updated the comment